.travis.yml

```diff
@@ -40,6 +52,14 @@ deploy:
       branch: master
       tags: false # Disable publishing for release tags
 
+  # Publishing release artifacts
+  - provider: script
+    skip_cleanup: true
+    script: scripts/publish-release
+    on:
+      branch: master
+      tags: true  
```

Comment:
Our standard for this is
`if: ((branch = master AND type = push) OR (tag IS present)) AND NOT fork`
so that snapshots get published on master build.

Reply:
For publishing snapshots uses a previous block:
https://github.com/playframework/play-soap/blob/master/.travis.yml#L35-L41

But we cat a merge them if you want 😄

Reply:
Ok, I will re-work it later.